pkgs/applications/misc/blender/default.nix

```diff
@@ -59,6 +60,7 @@ stdenv.mkDerivation rec {
       "-DWITH_PYTHON_INSTALL=OFF"
       "-DWITH_PYTHON_INSTALL_NUMPY=OFF"
     ]
+    ++ optional enableNumpy "-DPYTHON_NUMPY_PATH=${pythonPackages.numpy}/${python.sitePackages}"
```

Comment:
When building from git in the past this seemed to be mandatory, I'll try again when I can download the file.

Reply:
```
cc1plus: warning: /build/blender/extern/audaspace/PYTHON_NUMPY_PATH-NOTFOUND/numpy/core/include: No such file or directory [-Wmissing-include-dirs]
[ 91%] Building C object source/blender/makesrna/intern/CMakeFiles/makesrna.dir/rna_boid.c.o
/build/blender/extern/audaspace/bindings/python/PySound.cpp:70:10: fatal error: numpy/ndarrayobject.h: No such file or directory
 #include <numpy/ndarrayobject.h>
          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [extern/audaspace/CMakeFiles/audaspace-py.dir/build.make:154: extern/audaspace/CMakeFiles/audaspace-py.dir/bindings/python/PySound.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2594: extern/audaspace/CMakeFiles/audaspace-py.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```